Modules/TRD/src/TrackletsTask.cxx

```diff
@@ -172,14 +172,19 @@ void TrackletsTask::buildHistograms()
   mTrackletsPerTimeFrameCycled.reset(new TH1F("trackletspertimeframecycled", "Number of Tracklets per timeframe, this cycle;Tracklets in TimeFrame;Counts", 25000, 0, 500000));
   getObjectsManager()->startPublishing(mTrackletsPerTimeFrameCycled.get());
 
+  //=====================newly added lines=================
```

Comment:
This comment is too generic. Remove it or be more specific.

Reply:
Ok....